app/models/comment.rb

```diff
@@ -85,7 +85,7 @@ def check_for_spam
     includes(
       pseud: { user: [:roles, :block_of_current_user, :block_by_current_user, :preference] },
       parent: { work: [:pseuds, :users] }
-    )
+    ).merge(Pseud.with_attached_icon)
```

Comment:
I've fixed some of these, but to keep the PR manageable (it's already quite large), I'm going to skip
* Anything related to collections (we're moving them to Elasticsearch, so things involving them and challenges will likely change quite a bit)
* Inbox comments (since there's already a ticket to handle them)